pyproject.toml

```diff
@@ -22,7 +22,7 @@ fsspec = "^2024.6.1"
 polars-lts-cpu = "^1.16.0"
 pyarrow = "^18.1.0"
 boto3 = "~1.34.0"
-s3fs = "^2024.9.0"
+s3fs = { version = "^2024.9.0", extras = ["aiohttp=^3.11.10"] }
```

Comment:
Probably worth a comment about why `aiohttp` has to be pinned.

Reply:
> Probably worth a comment about why `aiohttp` has to be pinned.

Ok added comment.  That does bring up the fact that at some point the pinning needed (which we had to do for an api-commons snyk error, too) will no longer.  So figuring out a strat, or just needing to remember, to periodically review and see if we can remove these.